CHANGELOG.md

```diff
@@ -45,6 +45,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Migrate client transports to Apache HttpClient / Core 5.x ([#4459](https://github.com/opensearch-project/OpenSearch/pull/4459))
 - Change http code on create index API with bad input raising NotXContentException from 500 to 400 ([#4773](https://github.com/opensearch-project/OpenSearch/pull/4773))
 - Improve summary error message for invalid setting updates ([#4792](https://github.com/opensearch-project/OpenSearch/pull/4792))
+- Make Tracer injectable
```

Comment:
Please move it under `Unreleased 2.x` and add please link to the pull request (like all other changelog entries), thank you.

Reply:
@suranjay I think we can omit a changelog entry since this isn't really user facing functionality and is part of the bigger tracer functionality under development. I'll add the skip-changelog label.

Reply:
@reta My bad, this happened second time i guess. Will be careful next time.

Reply:
@andrross Done, remove entry from changelog in the latest commit